Panto/css/color.css

```diff
@@ -10,11 +10,20 @@
 
   --color-product-background: hsl(0, 0%, 97%);
   --color-product-nav: hsl(0, 0%, 93%);
-  --color-product-category: hsl(0, 0%, 55%);
-  --color-product-details: hsl(221, 63%, 14%);
+  --color-product-item-category: hsl(0, 0%, 55%);
+  --color-product-item-details: hsl(221, 63%, 14%);
+  --color-product-image-box: hsl(0, 0%, 98%);
+
+  --color-experiences-materials-behind-image: var(--color-product-background);
```

Comment:
I don't think this needs to be turned into a custom property but anyway you can still define it here